src/Composer/StudioPlugin.php

```diff
@@ -23,6 +25,16 @@ class StudioPlugin implements PluginInterface, EventSubscriberInterface
      */
     protected $io;
 
+    /**
+     * @var array
+     */
+    protected $pluggedCommands = [
```

Comment:
Looks like a constant array to me. Could use `const PLUGGED_COMMANDS = [...]` or `private const PLUGGED_COMMANDS = [...]`, depending on project's supported PHP versions rane.

@franzliedke, what the Studio's minimum supported PHP version is? It's not specified in `composer.json`.

Reply:
Studio requires symfony/console ^2.7, which requires php >= 5.3.9

Reply:
After testing the package with phpcompatibility/php-compatibility it turns out that the minimum PHP version is 5.4.
It seems also that Composer creates autoload_static.php file which requires PHP >= 5.5:
```
FILE: /path/to/studio/vendor/composer/autoload_static.php
----------------------------------------------------------------------------------------------------------
FOUND 3 ERRORS AFFECTING 3 LINES
----------------------------------------------------------------------------------------------------------
 16 | ERROR | Constant scalar expressions are not allowed in property declarations in PHP 5.5 or earlier.
 23 | ERROR | Constant scalar expressions are not allowed in property declarations in PHP 5.5 or earlier.
 61 | ERROR | The magic class constant ClassName::class was not available in PHP 5.4 or earlier
----------------------------------------------------------------------------------------------------------
```

Since this is exactly what it's about here, I think we can use constant scalar expressions.